bpf/bpf_host.c

```diff
@@ -91,7 +91,8 @@ static __always_inline bool identity_from_ipcache_ok(void)
 #ifdef ENABLE_IPV6
 static __always_inline __u32
 resolve_srcid_ipv6(struct __ctx_buff *ctx, struct ipv6hdr *ip6,
-		   __u32 srcid_from_proxy, const bool from_host)
+		   __u32 srcid_from_proxy, __u32 *sec_identity,
```

Comment:
```suggestion
		   __u32 srcid_from_ipcache, __u32 *sec_identity,
```

and delete below assignment.

Reply:
:+1: